components/button-cards/category-cards/category-card-dark/category-card-dark.config.yml

```diff
@@ -1,5 +1,5 @@
 status: 'wip'
-notes: 'Overview of Category Cards'
+notes: 'Overview of available colors'
```

Comment:
hm doesn't seem right?

Reply:
that was not part of what we were doing but we fixed it.